static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -192,6 +212,116 @@ function BreadcrumbItem({
   );
 }
 
+function WebVitalData({
+  replay,
+  frame,
+  expandPaths,
+  onInspectorExpanded,
+  onMouseEnter,
+  onMouseLeave,
+}: {
+  expandPaths: string[] | undefined;
+  frame: WebVitalFrame;
+  onInspectorExpanded: (path: string, expandedState: Record<string, boolean>) => void;
+  onMouseEnter: MouseCallback;
+  onMouseLeave: MouseCallback;
+  replay: ReplayReader | null;
+}) {
+  // TODO: remove test CLS data once SDK is merged and updated
+  const clsFrame = {
+    ...frame,
+    data: {
+      value: frame.data.value,
+      size: frame.data.size,
+      rating: frame.data.rating,
+      nodeIds: [frame.data.nodeIds, 333, 870],
+      attributes: [
+        {value: 0.0123, nodeIds: frame.data.nodeIds ?? [93]},
+        {value: 0.0345, nodeIds: [333, 870]},
+      ],
+    },
+  };
```

Comment:
to be removed once the sdk is updated

Reply:
Are older sdk versions sending data in this format? if so we'd need to continue to support it as there are customers out there on that older version